_posts/1100-09-01-AWS-installing-on-multiple-accounts.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2018-04-17
+title: Multiple AWS Accounts
+categories:
+    - amazon web service
+description: "A guide for installing Spinnaker on multiple AWS accounts from a single repo"
```

Comment:
What do you mean by "from a single repo"?

Reply:
My guess is the client meant he is using a single code repository to deploy.  I can see where you are going, I don't see the relevance here.  Removing.